.ci/create-changes-html.sh

```diff
@@ -19,7 +19,7 @@ echo '<script>hljs.highlightAll();</script>' >> CHANGES.html
 cat >> CHANGES.html << EOF
 <script>
 document.addEventListener('DOMContentLoaded', () => {
-const baseDocURL = 'https://sagemath.netlify.app'
+const baseDocURL = 'https://doc-develop--sagemath.netlify.app'
```

Comment:
Add a similar comment here like you did in src/sage_docbuild/conf.py?

Reply:
Right. Thanks.